src/hooks/airdrop/ws/useHandleWsGlobalEvent.ts

```diff
@@ -0,0 +1,16 @@
+import { useAirdropStore } from '@app/store/useAirdropStore';
+import { WebsocketEventNames, WebsocketGlobalEvent } from '@app/types/ws';
+
+export const useHandleWsGlobalEvent = () => {
+    return (event: string) => {
+        const eventParsed = JSON.parse(event) as WebsocketGlobalEvent;
+        switch (eventParsed.name) {
+            case WebsocketEventNames.X_SPACE_EVENT:
+                useAirdropStore.setState({ latestXSpaceEvent: eventParsed.data });
+                break;
+            default:
+                // eslint-disable-next-line no-console
+                console.log('Unknown global event', eventParsed);
```

Comment:
can you use `console.info` or `console.warn` for this one maybe? since we have rule overrides for that, then you don't need to disable next-line